go/common/sgx/pcs/cache_test.go

```diff
@@ -169,7 +201,8 @@ func TestTCBCache(t *testing.T) {
 	} {
 		t.Run(name, func(t *testing.T) {
 			fun(t, store, &tcbBundle)
-			_ = store.Delete([]byte(tcbCacheKey))
```

Comment:
Will this remain in the cache forever if not deleted in the new version?

Reply:
The old keys? Yeah, we can clear or migrate.

Reply:
Added migration.